From: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
Date: Wed, 1 Jun 2016 21:44:49 +0200
Subject: [PATCH] FIXME: Remove repo-mirror-cz.patch.

Upstream removed all mirrors but usa and cz, and changed
its default to mirror.fuel-infra.org in [1].

This allows us to drop patching of fuel-main/config.mk
with the Czech mirror.

If needed, USE_MIRROR=cz can be specified as an env var.

FIXME: Remove when Fuel@OPNFV pulls change and deals with it.

[1] https://github.com/openstack/fuel-main/commit/
    751d502cfe15d9c9df0ee89530ac3b0b73aa1638

Signed-off-by: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
---
 build/Makefile             |  2 --
 build/repo-mirror-cz.patch | 19 -------------------
 2 files changed, 21 deletions(-)
 delete mode 100644 build/repo-mirror-cz.patch

diff --git a/build/Makefile b/build/Makefile
index e84edcd..211177d 100644
--- a/build/Makefile
+++ b/build/Makefile
@@ -106,8 +106,6 @@ $(ISOCACHE):
 	fi
 	cd /tmp/fuel-main && git checkout $(FUEL_MAIN_TAG)
 	@echo "fuel" `git -C /tmp/fuel-main show | grep commit | head -1 | cut -d " " -f2` >> $(VERSION_FILE)
-	# Patch for using the Czech Fuel mirror
-	cd /tmp/fuel-main && patch -p0 < $(TOPDIR)/repo-mirror-cz.patch
 	# Remove Docker optimizations, otherwise multistrap will fail during
 	# Fuel build.
 	sudo rm -f /etc/apt/apt.conf.d/docker*
diff --git a/build/repo-mirror-cz.patch b/build/repo-mirror-cz.patch
deleted file mode 100644
index aa8eaf9..0000000
--- a/build/repo-mirror-cz.patch
+++ /dev/null
@@ -1,19 +0,0 @@
-*** config.mk.orig	Thu Jan  7 23:30:38 2016
---- config.mk	Thu Jan  7 23:32:36 2016
-***************
-*** 153,159 ****
-  # 'msk', 'srt', 'usa', 'hrk', 'cz'
-  # Setting any other value or removing of this variable will cause
-  # download of all the packages directly from internet
-! USE_MIRROR?=ext
-  
-  ifeq ($(USE_MIRROR),ext)
-  MIRROR_FUEL?=http://mirror.fuel-infra.org/mos-repos/centos/$(PRODUCT_NAME)$(PRODUCT_VERSION)-centos$(CENTOS_MAJOR)-fuel/os/x86_64/
---- 153,159 ----
-  # 'msk', 'srt', 'usa', 'hrk', 'cz'
-  # Setting any other value or removing of this variable will cause
-  # download of all the packages directly from internet
-! USE_MIRROR?=cz
-  
-  ifeq ($(USE_MIRROR),ext)
-  MIRROR_FUEL?=http://mirror.fuel-infra.org/mos-repos/centos/$(PRODUCT_NAME)$(PRODUCT_VERSION)-centos$(CENTOS_MAJOR)-fuel/os/x86_64/
